### PR TITLE
Remove line through style in embedded profile description

### PR DESCRIPTION
**Pull request by kpet**
If this was supposed to mean something, it should be clearly documented.

Signed-off-by: Kevin Petit <kevin.petit@arm.com>

### CODE REVIEW

**Comment by mkinsner** [comment]
I believe that these embedded profile restrictions were in place up to and including OpenCL 1.2, and then removed with the strikethrough in OpenCL 2.0, making embedded profile implementations build slightly more functionality.

By removing the strikethrough, I think we're re-adding more flexibility for embedded profile implementations in 2.0 and onward, which probably isn't the goal here.

The strikethrough formatting looks broken in the rendered 2.2 spec - probably a bug from format conversion.  2.0 and 2.1 look correct.

If we're changing something, the text should just be deleted I think.

**Comment by kpet** [comment]
Thanks Mike! I had a suspicion this was more than a typo. 

>By removing the strikethrough, I think we're re-adding more flexibility for embedded profile implementations in 2.0 and onward, which probably isn't the goal here.

Yup, not the goal. I just stumbled upon this as part of #202 and wanted to push for clarity.

>If we're changing something, the text should just be deleted I think.

I don't think we can do that given that the spec now describes all versions of OpenCL and it is still possible for a 1.x implementation to use the flexibility. Maybe that got missed during spec unification. @alycm Do you know more?

BTW, the OpenCL C spec also has a few bits of text with strikethrough. We probably want to clarify this as well.

**Comment by alycm** [comment]
I had seen these strikethroughs before, initially assume it was a formatting error and then saw that it was clearly on purpose so left it.

Yes, the unified spec should describe the difference between <2.0 and >=2.0.   Yes, this got missed.  I'm actually going through this right now, so happy to add this bit.

**Comment by kpet** [comment]
Thanks! I'll close this PR after reviewing yours then.

File: api/embedded_profile.asciidoc [hunk 1]
```diff
@@ -42,15 +41,15 @@ embedded profile implementation.
 {clCreateImage} will work as defined by the OpenCL specification.
 The `image3d_t` data type can be used in a kernel(s).
 --
-  . [line-through]#Support for 2D image array writes is optional.
+  . Support for 2D image array writes is optional.
     If the *cles_khr_2d_image_array_writes* extension is supported by the
-    embedded profile, writes to 2D image arrays are supported.#
-  . [line-through]#Image and image arrays created with an
+    embedded profile, writes to 2D image arrays are supported.
```
[the remark this answers is not in the document]
Comment (alycm) [reply api/embedded_profile.asciidoc]: Unrelated to this PR, but related to this comment: The embedded profile also uses the **cles_khr_int64** extension name to declare whether long/ulong are supported on a device.

**Comment by alycm** [comment]
The <2.0 vs >=2.0 difference is described in this PR now: https://github.com/KhronosGroup/OpenCL-Docs/pull/209

**Comment by kpet** [comment]
Now superseded by #209. Closing.

**Pull request closed without merging**